app/packages/core/src/components/ColorModal/ColorFooter.tsx

```diff
@@ -38,6 +38,11 @@ const ColorFooter: React.FC = () => {
     return true;
   }, [savedSettings]);
 
+  // set to be true only for teams
+  const isTeams = useRecoilValue(fos.compactLayout);
```

Comment:
This is logic/naming something we should avoid. `compactLayout` means `isTeams` by coincidence only